osquery/events/linux/auditdfim.h

```diff
@@ -0,0 +1,129 @@
+#pragma once
+
+#include "osquery/events/linux/auditdnetlink.h"
+#include <osquery/events.h>
```

Comment:
nit, prefer to order includes:
1. C
2. C/OS-specific
3. C++
4. C++ third party libraries
5. Project global, aka the top-level `./include/osquery` using the `<>` search order.
6. Project local, so `""` search order.

And place a `\n\n` between each group so clang-format does not apply a total alphanumeric ordering.